tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/symbols/TypeSymbolTest.java

```diff
@@ -78,48 +80,51 @@ public void setup() {
     }
 
     @Test(dataProvider = "PosProvider")
-    public void testTypeSymbolLookup(int line, int col, TypeDescKind typeKind, String signature) {
+    public void testTypeSymbolLookup(int line, int col, TypeDescKind typeKind, String signature, LineRange lineRange) {
         TypeSymbol type = assertBasicsAndGetType(line, col, typeKind, signature);
         assertTrue(type.getName().isEmpty());
-        assertTrue(type.getLocation().isEmpty());
+        assertEquals(type.getLocation().map(Location::lineRange).orElse(null), lineRange);
```

Comment:
let's have a check for `null` values too
```
if (lineRange == null) {
    assertTrue(type.getLocation().isEmpty());
} else {
    assertEquals(type.getLocation(), lineRange);
}
.. 
```

Reply:
We are doing the same thing here, right?

Reply:
Oh yeah. This is okay.